apps/antalmanac/src/components/RightPane/SectionTable/CourseInfoButton.tsx

```diff
@@ -87,7 +87,20 @@ function CourseInfoButton({
                     }
                 }}
             >
-                {text}
+                <span style={{ display: 'flex', gap: 4 }}>
+                    {icon}
```

Comment:
question/issue: can we continue passing in icon as `startIcon`? Why are we moving this into the children?

Reply:
startIcon auto adds margin right, making it not centered